storybook/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -19,7 +19,14 @@ const meta = {
   title: 'Components/Containers/Accordion',
   component: Accordion,
   args: {
-    headingLevel: 1,
+    headingLevel: 3,
```

Comment:
It seems to default to level 4 now, I think setting the label to `undefined: 1` is doing something weird.